crates/fixt/Cargo.toml

```diff
@@ -5,25 +5,25 @@ description = "minimum viable fixtures"
 license = "Apache-2.0"
 homepage = "https://github.com/holochain/holochain"
 documentation = "https://docs.rs/fixt"
-authors = [ "Holochain Core Dev Team <devcore@holochain.org>" ]
-keywords = [ "holochain", "holo", "test", "fixture" ]
-categories = [ "development-tools::testing" ]
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
```

Comment:
What are you using to format toml? And can we run it on CI? :)

Reply:
I use taplo with default settings https://github.com/tamasfe/taplo

I guess we can run it on CI.